### PR TITLE
Create Synchronizing Project for the web tasks with Planner and To Do

### PR DESCRIPTION
**Pull request by SOURSH**
This change is part of "Project Planner Integration" work. Roadmap id - https://www.microsoft.com/en-us/microsoft-365/roadmap?rtc=1&filters=Planner&searchterms=382207

### CODE REVIEW

**Comment by SOURSH** [comment]
I want to attach this image along with the text, couldnt find a way to do that
ATM flag

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+An admin can enable or disable Project for the web task synchronization with Planner and To Do for all users in their organization through the Project settings page in the Microsoft 365 admin center. 
+
+Steps to Enable or Disable Project for the web Task Syncing: 
+
+1. In the Microsoft 365 admin center, expand the navigation menu, select Settings, and then select Org Settings. 
+2. Select Project. 
+3. On the Project settings page, select or deselect Turn on Assigned to me view syncing for your organization, and then select Save. 
+
+
```
Comment (SOURSH) [inline ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:15]: ATM flag

**Comment by SOURSH** [comment]
@jacktremper can you please review and merge this, I dont have the write access, hence the request.

**Comment by jacktremper** [comment]
@jenzamora and @mscharlo to review and sign off

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Synchronizing Project for the web tasks with Planner and To Do 
+
+The Assigned to me view in Planner and To Do displays all Microsoft 365 tasks assigned to a user. By default, syncing of Project for the web tasks with Planner and To Do is enabled for all tenants. However, if your Project for the web and Planner are in two different data boundaries and you do not want Project for the web data to cross the data boundary, you can disable this feature for your tenant. This synchronization can only be enabled or disabled at the tenant level, not at the user level.
```
Comment (mscharlo) [inline ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:3]:
```suggestion
By default, Project for the web tasks with Planner and To Do is enabled for all tenants. This means that the Assigned to me view in Planner and To Do will display all Microsoft 365 tasks assigned to a user. 

However, there may be instances when disabling this synchronization at the tenant level would be helpful, such as if you require your data for Project for web and Planner to exist in two different data boundaries. 
```

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Synchronizing Project for the web tasks with Planner and To Do 
+
+The Assigned to me view in Planner and To Do displays all Microsoft 365 tasks assigned to a user. By default, syncing of Project for the web tasks with Planner and To Do is enabled for all tenants. However, if your Project for the web and Planner are in two different data boundaries and you do not want Project for the web data to cross the data boundary, you can disable this feature for your tenant. This synchronization can only be enabled or disabled at the tenant level, not at the user level.  
+
+How admins can control syncing of Project tasks with Planner and To Do
```
Comment (mscharlo) [inline ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:5]:
```suggestion
## Admin controls to enable or disable synchronization
```

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Steps to Enable or Disable Project for the web Task Syncing: 
+
+1. In the Microsoft 365 admin center, expand the navigation menu, select Settings, and then select Org Settings. 
+2. Select Project. 
+3. On the Project settings page, select or deselect Turn on Assigned to me view syncing for your organization, and then select Save. 
+
+
+
+Please note that Assigned to me view syncing (i.e. Project task syncing with Planner and To Do) can only be enabled if “Project for the web” has been enabled for the tenant. If the administrator decides to disable “Project for the web” for the tenant, Assigned to me view syncing will also be disabled and no new data will be copied to Planner and To Do. However, previously synchronized data will not be deleted until the Assigned to me view syncing is explicitly disabled for the tenant.
```
Comment (mscharlo) [inline ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:17]:
```suggestion
> [!NOTE]
>]In order for Project for the web task with Planner and To Do tasks to synchronize, Project for the web must be enabled for the tenant. If the administrator decides to disable it, tasks won't be synchronized and no new data will be copied to Planner and To Do. Note that previously stored data won't be deleted until the Assigned to me view syncing is explicitly disabled for the tenant. 
```

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do (hunk 1 above)
Comment (mscharlo) [inline ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:15]: Although an image is helpful to English speaking users, we don't have a translation service for images yet. For this reason, I think it's probably a good idea to leave this out (as much as I am a fan of images). If there are other details we could put in the instructions that might make the details of this image more clear, feel free to add!

**Review by mscharlo — COMMENTED** [review]
My question is where do you want this to show up in the table of contents and are there any articles that we could link to at the bottom for a kind of "learn more" for users that you can think of? This article also needs some metadata added to the top so it can be properly in our system (I am new so learning what this needs to be, but just thought I would mention, I will suggest it in a second round of edits once I know what it is supposed to be).

File: ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do (hunk 1 above)
Comment (SOURSH) [reply ProjectWeb/Synchronizing Project for the web tasks with Planner and To Do:15]: @mscharlo we already have an image for the other two similar features, hence to maintain consistency, we need to have added the image included

**Comment by SOURSH** [comment]
> My question is where do you want this to show up in the table of contents and are there any articles that we could link to at the bottom for a kind of "learn more" for users that you can think of? This article also needs some metadata added to the top so it can be properly in our system (I am new so learning what this needs to be, but just thought I would mention, I will suggest it in a second round of edits once I know what it is supposed to be).

**Comment by SOURSH** [comment]
@mscharlo The content has to show up between "Turn Project for the web or Roadmap on or off for users in your organization"  and "Project architecture overview"
image

**Comment by SOURSH** [comment]
@mscharlo I dont have the write access, can you please merge this PR and publish the content?

File: ProjectWeb/synchronizing-project-for-the-web-tasks-with-planner-and-todo.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+An admin can enable or disable Project for the web task synchronization with Planner and To Do for all users in their organization through the Project settings page in the Microsoft 365 admin center. 
+
+Use the following steps to Enable or Disable Project for the web Task Syncing: 
+
+1. In the Microsoft 365 admin center, expand the navigation menu, select **Settings**, and then select **Org Settings**. 
+2. Select **Project**. 
+3. On the Project settings page, select or clear the checkbox **Turn on Assigned to me view syncing for your organization**, and then select **Save**. 
+
+> In order for Project for the web task to synchronize with Planner and To Do tasks, Project for the web must be enabled for the tenant. If the administrator decides to disable it, tasks won't be synchronized and no new data will be copied to Planner and To Do. Note that previously stored data won't be deleted until the Assigned to me view syncing is explicitly disabled for the tenant.
```
Comment (mscharlo) [inline ProjectWeb/synchronizing-project-for-the-web-tasks-with-planner-and-todo.md:30]: I'm pretty sure this format has to be here for it to render properly
```suggestion
> [!NOTE]
> In order for Project for the web task to synchronize with Planner and To Do tasks, Project for the web must be enabled for the tenant. If the administrator decides to disable it, tasks won't be synchronized and no new data will be copied to Planner and To Do. Note that previously stored data won't be deleted until the Assigned to me view syncing is explicitly disabled for the tenant. 
```

**Comment by mscharlo** [comment]
@jenzamora please tag for publishing

**Comment by mscharlo** [comment]
Closing this pr as I had @soursh re-submit a pr on the OfficeDocs-ProjectServer-pr repo instead. Thank you!

**Comment by SOURSH** [comment]
[like]  Sourabh Sharma reacted to your message:
________________________________

**Pull request closed without merging**